readme.md

```diff
@@ -290,8 +289,9 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [InKeV: In-Kernel Distributed Network Virtualization for DCN](https://github.com/iovisor/bpf-docs/blob/master/university/sigcomm-ccr-InKev-2016.pdf)
 - [gobpf - utilizing eBPF from Go](https://fosdem.org/2017/schedule/event/go_bpf/) - A library to create, load and use eBPF programs from Go.
 - [ply](https://wkz.github.io/ply/) - A small but flexible open source dynamic tracer for Linux, with features similar to the bcc tools, but with a simpler language inspired by awk and DTrace.
-- [bpftrace](https://github.com/iovisor/bpftrace) - A tool for tracing, again with its own DSL. It is flexible enough to be envisioned as a Linux replacement for DTrace and SystemTap. More information also found on [bpftrace.org](https://bpftrace.org/).
-- [bpftrace Cheat Sheet](https://www.brendangregg.com/BPF/bpftrace-cheat-sheet.html) - bpftrace cheat sheet that you can print out for reference, and is from the [bpftrace programming](https://www.brendangregg.com/ebpf.html#bpftraceprogramming) section of Brendan's [eBPF Tools](https://www.brendangregg.com/ebpf.html) page. 
+- [bpftrace](https://bpftrace.org/) - A tool for tracing with its own high-level tracing language. It is flexible enough to be envisioned as a Linux replacement for DTrace and SystemTap.
+  - [Linux Extended BPF (eBPF) Tracing Tools](https://www.brendangregg.com/ebpf.html) - A in-depth collection of information around examples of performance analysis tools using eBPF.
```

Comment:
I thought you'd keep the other link (with the section anchor). OK to keep this one, but then let's find another place for it? It's good enough that it can go under the generic articles and presentations, I'd think.